atmo/clusters/models.py

```diff
@@ -36,6 +40,13 @@ class Cluster(models.Model):
         max_length=50, blank=True, null=True,
         help_text="AWS cluster/jobflow ID for the cluster, used for cluster management."
     )
+
+    emr_release = models.CharField(
+        max_length=50, choices=zip(*(EMR_RELEASES,)*2), default=EMR_RELEASES[-1],
```

Comment:
`zip` returns a `zip` object on Python 3, not sure if the choices parameter handles that nicely. wrapping it in a `list` would appease my mind.

Side note: did we ever want to make atmo Python 3 btw?